v3/tutorials/07-add-a-pallet/index.mdx

```diff
@@ -395,7 +395,7 @@ To return the information stored for Alice:
 
 1. Select `nicks` from the list of pallets available to query.
 
-1. Select the [`nameOf`](/rustdocs/latest/pallet_nicks/pallet/enum.Call.html#variant.set_name).
+1. Select the [`nameOf`](/rustdocs/latest/src/pallet_nicks/lib.rs.html#113).
```

Comment:
It's preferable to point to the rustdocs not the source hosted by rustdocs, or the source instead on github.

Reply:
Agree. But I wasn't able to find a reference for the `nameOf` Storage there =/.
Right now the link points to [enum.Call.html#variant.set_name](https://docs.substrate.io/rustdocs/latest/pallet_nicks/pallet/enum.Call.html#variant.set_name) so maybe we should keep it as it is.